eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -53,6 +53,7 @@ jobs:
           /p:SDKType=${{ parameters.SDKType }}
           /p:ServiceDirectory=${{ parameters.ServiceToTest }}
           /p:IncludeSrc=false /p:IncludeSamples=false /p:IncludePerf=false /p:IncludeStress=false
+          /p:RunApiCompat=false /p:InheritDocEnabled=false
```

Comment:
I assume these are already disabled for test projects but this disables them for all the library projects just for the test run in CI for the.